example/lib/movas/observables/feed_items_observable.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:convert';
+
+import 'package:flutter/foundation.dart';
+
+import 'package:movas_example/movas/services/responses/feed_items.dart';
+
+import 'feed_item_observable.dart';
+
+class FeedItemsO {
```

Comment:
we try to separate models from observables.

models are classes that are serialized from services.

although everything could work just as well by having models provided as observables, it is good practice to separate that logic.

you can make observable that just wraps model. that is alright.

Reply:
so no need to reparse it, just wrap the response